.env.example

```diff
@@ -11,3 +11,9 @@ SOCIALHOME_DOMAIN=127.0.0.1:8000
 SOCIALHOME_HTTPS=False
 
 MOCHA_TESTS=True
+
+# Set these variables for Docker Postgres (be sure the DATABASE_URL matches)
+POSTGRES_USER=socialhome
```

Comment:
As discussed, these variables should be imho defined in `docker-compose.yml` (which is missing btw). Otherwise we have database settings twice, and really there is no reason for developers normally to change these.

Reply:
:heavy_check_mark: